### DISCUSSION TITLE
Welcome to Dcm2Bids Discussions!

### DISCUSSION THREAD

**Discussion by arnaudbore** [UNFmontreal/Dcm2Bids#123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arnaudbore** [comment]
Hi everybody,

## Quick message from Montreal :canada: 
First of all, thank you for being part of the dcm2bids community :heart: :tada: :raised_hands:  I love this project and love that it makes something that could be painful into something so easy to achieve. Let's be honest converting your DICOM to a BIDS structure should not be a significant part of your research project. :stuck_out_tongue_winking_eye:

## Few words about me
My name is Arnaud, I'm the main maintainer of dcm2bids :confetti_ball: 

I work as a research assistant for [UNF](http://unf-montreal.ca/) and the [SCIL](http://scil.dinf.usherbrooke.ca/en/). I've got a master from Sherbrooke University :canada:  in diffusion MRI. I like to dev (mostly in python :snake:), more generally I'm a bug-fixer :bug: but mostly I love helping people to do better open science.

See you soon :v:

**Comment by SamGuay** [comment]
Hello everyone :wave: !

I am Sam, a PhD student in Psychology - Cognitive Neuroscience @ [University of Montreal](https://umontreal.ca) using ~~too~~ many neuroimaging modalities to study the effects of repetitive head impacts on the brain of active and retired athletes.

> Let's be honest converting your DICOM to a BIDS structure should not be a significant part of your research project.

I agree 100 % with the above statement. As an end-user, the first time I tried to convert my images wasn't pretty :grimacing:. I had tried many solutions, and `dcm2bids` stuck for its ease of use and simplicity. _Do one thing and do it well as [they say](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well)_ :eyes:.

Now that I am used to it, I slowly build my confidence to eventually add a couple of features I would like to see implemented. I also like writing documentation to make tools even more accessible, so you may see me constantly improving the doc and adding tutorials as the software improves.

Looking forward to collaborating with all of you :technologist:
